response_operations_ui/views/messages.py

```diff
@@ -127,27 +127,28 @@ def view_conversation(thread_id):
     try:
         messages = [_thread_refine(message) for message in message_controllers.get_conversation(thread_id)['messages']]
     except KeyError as ex:
-        # TODO change exception message
         logger.exception("A key error occurred")
         raise ApiError(ex)
+    subject = _get_messages_subject(messages)
     breadcrumbs = [
         {"title": "Messages", "link": "/messages"},
-        {"title": f'{_get_messages_subject(messages)}'}
+        {"title": subject}
     ]
 
-    return render_template("conversation-view.html", breadcrumbs=breadcrumbs, messages=messages)
+    return render_template("conversation-view.html", breadcrumbs=breadcrumbs, messages=messages, subject=subject)
 
 
 def _get_messages_subject(messages):
     try:
-        return messages[0].get("subject")
-    except (IndexError, AttributeError):
-        return 'Unavailable'
+        return messages[0]["subject"]
+    except KeyError:
+        logger.exception("Failed to retrieve RU ref from message")
+        return "Unavailable"
```

Comment:
It would be better if we returned None and had the fallback text in the template

Reply:
got fallback text in html